Automattic/wp-calypso
# A4A: Implement migration offer banner v2 based on the new design.

This pull request implements the new Migration Offer card based on the updated design.
<img width="1351" alt="Screenshot 2024-07-24 at 9 02 06 PM" src="https://github.com/user-attachments/assets/51d7fc01-9e8e-43e2-9c6c-010a12241f55">

Closes https://github.com/Automattic/jetpack-genesis/issues/435

## Proposed Changes

* Implement the `MigrationOfferV2` component based on the new design. Note that the component does not use the Foldable card component from Calypso as the new design has different spacing, which makes it hard to achieve with the existing component.

## Why are these changes being made?

*

## Testing Instructions

* Use the A4A live link below and go to the `marketplace/hosting` page.
* Confirm that you see the new Migration offer banner.
* Test that the functionality works as expected (links, view toggle, etc.)

## Pre-merge Checklist
